### DISCUSSION TITLE
Help with uploading the project to the hosting

### DISCUSSION THREAD

**Discussion by bahmN** [def-studio/telegraph#445] [Q&A]
Initially, the project was created on a local PC, using Docker + Ngrok. Everything worked correctly, notifications were received and processed, respectively, responses were also sent to the bot. I uploaded the project to the hosting, the notifications from telegram stopped coming and, accordingly, they are not processed in any way. Tell me, please, how was it necessary to upload the project to the hosting?

**Comment by fabio-ivona** [comment]
Hi @bahmN we don't offer such a service on github, for consulting and mentorship please use the [mentorship one-time sponsorship tier](https://github.com/sponsors/defstudio/sponsorships?sponsor=fabio-ivona&tier_id=367017)
